### PR TITLE
4.22 Upgrade

### PR DESCRIPTION
**Pull request by joshuahuburn**
Upgrade the current master to build with 4.22

### CODE REVIEW

**Comment by oblm** [comment]
imma build and playtest this

File: Game/Source/GDKShooter/Private/Weapons/Weapon.cpp [hunk 1]
```diff
@@ -198,6 +197,7 @@ void AWeapon::Tick(float DeltaTime)
 			}
 		}
 	}
+	UE_LOG(LogGDK, Verbose, TEXT("TEST"));
```
Comment (improbable-valentyn) [inline Game/Source/GDKShooter/Private/Weapons/Weapon.cpp]:
```suggestion
```

File: spatial/spatialos.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "your_project_name_here",
   "project_version": "0.0.1",
-  "sdk_version": "13.7.1",
+  "sdk_version": "13.8.1",
```
Comment (improbable-valentyn) [inline spatial/spatialos.json]: Shouldn't we be using 13.8.2?

Comment (joshuahuburn) [reply spatial/spatialos.json]: That's not what the version is set to in the SDK.version. These need to be in sync now due to spatiald and spot.

**Comment by joshuahuburn** [comment]
Closing this as master has been repointed.

**Pull request closed without merging**